concepts/predictions.md

```diff
@@ -1,6 +1,6 @@
 # Predictions
 
-A prediction in Cradle results from an application of an active, trained [Model](models.md) to a [Document](documents.md). It supplies the extracted data together with confidence values representing the probability that the extracted data is correct.
+A prediction in Cradle results from an application of an active, trained [Model](models.md) on a [Document](documents.md). It supplies the extracted data together with confidence values representing the probability that the extracted data onis correct.
```

Comment:
In my head, "apply on" is a more physical action, like "apply lotion on your skin", while "apply to" is more methodological, e.g. "we apply the method to the problem".

```suggestion
A prediction in Cradle results from an application of an active, trained [Model](models.md) to a [Document](documents.md). It supplies the extracted data together with confidence values representing the probability that the extracted data is correct.
```